server/src/test/java/com/objectcomputing/checkins/services/reports/FileServicesImplReplacement.java

```diff
@@ -0,0 +1,59 @@
+package com.objectcomputing.checkins.services.reports;
+
+/*************************************************************************
+ *
+ * This class is here due to the fact that the ReportDataController now
+ * references the FileServices.  The real FileServicesImpl requires the
+ * GoogleApiAccess class that does not exist during testing.
+ *
+ * This replacement class does not require that and can help us test the
+ * output of the MarkdownGeneration class.
+ *
+ ************************************************************************/
+
+import com.objectcomputing.checkins.services.file.FileInfoDTO;
+import com.objectcomputing.checkins.services.file.FileServices;
+import com.objectcomputing.checkins.services.file.FileServicesImpl;
+
+import io.micronaut.http.multipart.CompletedFileUpload;
+
+import java.io.File;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.UUID;
+
+import jakarta.inject.Singleton;
+import io.micronaut.context.env.Environment;
+import io.micronaut.context.annotation.Replaces;
+import io.micronaut.context.annotation.Requires;
+
+@Singleton
+@Replaces(FileServicesImpl.class)
+@Requires(env = Environment.TEST)
```

Comment:
That's the magic!

Reply:
Yep!  It came to me one morning to do it like we do the email sender!